pkgs/top-level/all-packages.nix

```diff
@@ -39895,7 +39895,9 @@ with pkgs;
 
   vivisect = with python3Packages; toPythonApplication (vivisect.override { withGui = true; });
 
-  vokoscreen = libsForQt5.callPackage ../applications/video/vokoscreen { };
+  vokoscreen = libsForQt5.callPackage ../applications/video/vokoscreen {
+    ffmpeg = ffmpeg_4-full;
```

Comment:
Is it really requiring `ffmpeg_4-full` instead of `ffmpeg-full`?

Reply:
Good point. I've just confirmed the change works with both ffmpeg 4 and 5, and changed the params to use `ffmpeg-full`.